### ISSUE TITLE
hyperparameter of VTAB-1k

### ISSUE THREAD

**Issue by L-JunJie** [runtsang/VFPT#2]
Dear author, could you please provide the hyperparameters used for your paper on the VTAB dataset, including learning rate, weight decay, and prompt length?

**Comment by runtsang** [comment]
https://docs.google.com/spreadsheets/d/1ojpqLU9Smf1NZHDGAM0HsWmGdJ1BOKAKxAJc7JQKa6Y/edit?usp=sharing
hope this helps!
If you find it helpful, feel free to cite our paper and start our repo.
have a good one! ツ
